src/dotenv/main.py

```diff
@@ -111,13 +111,14 @@ def get_key(
     dotenv_path: Union[str, os.PathLike],
     key_to_get: str,
     encoding: Optional[str] = "utf-8",
+    verbose: bool = True,
 ) -> Optional[str]:
     """
     Get the value of a given key from the given .env.
 
     Returns `None` if the key isn't found or doesn't have a value.
     """
-    return DotEnv(dotenv_path, verbose=True, encoding=encoding).get(key_to_get)
+    return DotEnv(dotenv_path, verbose=verbose, encoding=encoding).get(key_to_get)
```

Comment:
Added this flag because when invoked from `cli`, we may check multiple files, and if one includes the key, we don't want to emit n warnings.

Therefore the cli can call it with `verbose = False` and check whether it should warn or not.

Reply:
Doesn't seem needed anymore, as `cli.get` doesn't invoke this anymore.